front/src/common/BootstrapSNCF/CardSNCF/CardSNCF.tsx

```diff
@@ -5,7 +5,7 @@ interface CardProps {
   link: string;
   img: string;
   title: string;
-  disabledLink: boolean;
+  disabledLink?: boolean;
```

Comment:
Just one question: does this change have anything to do with the commit name?

Reply:
It's the "Make some component properties optional to reflect current usage." part :) 

The JavaScript files were constructing a `CardProps` object without a `disabledLink` property. This happens to work fine, but the type definition did not allow this. Previously the TypeScript compiler wouldn't complain because it doesn't check pure JavaScript files, but now that these are TypeScript, it errors out.